packages/web-client/src/webdav/getFileContents.ts

```diff
@@ -1,13 +1,16 @@
+import { Headers } from 'webdav'
 import { urlJoin } from '../utils'
 import { isPublicSpaceResource, SpaceResource } from '../helpers'
 import { WebDavOptions } from './types'
+import { buildPublicLinkAuthHeader, DAV } from './client'
+import { HttpError } from '../errors'
 
 export type GetFileContentsResponse = {
   body: any
   [key: string]: any
 }
 
-export const GetFileContentsFactory = ({ sdk }: WebDavOptions) => {
+export const GetFileContentsFactory = (dav: DAV, { clientService }: WebDavOptions) => {
```

Comment:
This is technically not webdav, we just use the webdav URL to get the file. Hence I decided to use the `ClientService` here instead of implementing a `get` method in our `DAV` abstraction.